agent/stats/engine_integ_test.go

```diff
@@ -340,6 +340,86 @@ func TestStatsEngineWithNewContainers(t *testing.T) {
 	validateEmptyTaskHealthMetrics(t, engine)
 }
 
+func TestStatsEngineWithNewContainersWithPolling(t *testing.T) {
+	// additional config fields to use polling instead of stream
+	cfg.PollMetrics = true
+	cfg.PollingMetricsWaitDuration = 1 * time.Second
```

Comment:
the cfg object is shared by many tests in this package.. we probably should set the value back if we are changing it for this test?

Reply:
I tried to clean up how cfg is shared by all integ tests, but it's a bigger refactoring job. I'll work on it separately, but for this one I reset it to false/default duration at the end of the test.